docs/configure-emulator.md

```diff
@@ -0,0 +1,178 @@
+```
+
+After initialization, the home directory will have the following structure:
+
+```shell
+$ ls <path>
+  ├── covd.conf # Covd-specific configuration file.
+  ├── logs      # Covd logs
+```
+
+### 4.2. Configure the covenant emulator
+
+As you have already set up the covenant signer, you can now configure the covenant 
+emulator to use it.
+
+Use the following parameters to configure the `covd.conf` file.
+
+```
+# The interval between each query for pending BTC delegations
+QueryInterval = 15s
+
+# The maximum number of delegations that the covd processes each time
+DelegationLimit = 100
+
+# Bitcoin network to run on
+BitcoinNetwork = simnet
+
+# Babylon specific parameters
+
+# Babylon chain ID
+ChainID = bbn-test-5
+
+# Babylon node RPC endpoint
+RPCAddr = https://rpc-euphrates.devnet.babylonlabs.io:443
+
+# Babylon node gRPC endpoint
+GRPCAddr = https://rpc-euphrates.devnet.babylonlabs.io:443
+
+# Name of the key in the keyring to use for signing transactions
+Key = covenant-key
+
+# Type of keyring to use,
+# supported backends - (os|file|kwallet|pass|test|memory)
+# ref https://docs.cosmos.network/v0.46/run-node/keyring.html#available-backends-for-the-keyring
+KeyringBackend = test
+
+[remotesigner]
+; URL of the remote signer
+URL = http://127.0.0.1:9792
+
+; client when making requests to the remote signer
+Timeout = 2s
+
+; if true, covenant will use the remote signer to sign transactions
+RemoteSignerEnabled = true
+```
+
+Ensure that the covenant signer is running and unlocked before proceeding 
+otherwise you will be unable to run the emulator.
+
+## 5. Generate key pairs
+
+The covenant emulator daemon requires the existence of a keyring that signs
+signatures and interacts with Babylon. Use the following command to generate the
+key:
+
+```bash
+$ covd create-key --key-name covenant-key --chain-id chain-test
+{
+    "name": "covenant-key",
+    "public-key": "9bd5baaba3d3fb5a8bcb8c2995c51793e14a1e32f1665cade168f638e3b15538"
+}
+```
+
+After executing the above command, the key name will be saved in the config file
+created in [step](#configuration).
+Note that the `public-key` in the output should be used as one of the inputs of
+the genesis of the Babylon chain.
+Also, this key will be used to pay for the fees due to the daemon submitting 
+signatures to Babylon.
+
+## 6. Start the emulator daemon
+
+You can start the covenant emulator daemon using the following command:
+
+```bash
+$ covd start
+2024-01-05T05:59:09.429615Z	info	Starting Covenant Emulator
+2024-01-05T05:59:09.429713Z	info	Covenant Emulator Daemon is fully active!
+```
+
+All the available CLI options can be viewed using the `--help` flag. These
+options can also be set in the configuration file.
+
+Next you will need to unlock the key and sign transactions. Please refer to the 
+[covenant signer setup guide](covenant-signer/README.md#using-the-covenant-signer-for-signing-transactions) 
+to unlock the key and sign any transactions that are needed.
```

Comment:
```suggestion
```



This does not relate to the emulator daemon, it has been covered by the other guide.